Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -152,19 +134,6 @@ - (instancetype)init
     _selectionGranularity = WKSelectionGranularityDynamic;
 #endif // PLATFORM(IOS_FAMILY)
 
-    _mediaContentTypesRequiringHardwareSupport = @"";
-
-#if ENABLE(APPLE_PAY)
-    _applePayEnabled = DEFAULT_VALUE_FOR_ApplePayEnabled;
-#endif
-
-#if ENABLE(APP_HIGHLIGHTS)
-    _appHighlightsEnabled = DEFAULT_VALUE_FOR_AppHighlightsEnabled;
-#endif
-
-    _sampledPageTopColorMaxDifference = DEFAULT_VALUE_FOR_SampledPageTopColorMaxDifference;
-    _sampledPageTopColorMinHeight = DEFAULT_VALUE_FOR_SampledPageTopColorMinHeight;
```

Comment:
`#import "WebPreferencesDefaultValues.h"` can probably be removed from this file now.